src/swarm/neo/protocol/connect/NodeConnect.d

```diff
@@ -185,6 +189,7 @@ class NodeConnect
 
                     // Authentication successful.
                     success = true;
+                    this.client_name.copy(client_name);
```

Comment:
Should you clean `this.client_name` somewhere? Not sure how feasible this scenario is:

1. Old client connects and authenticates
2. The client disconnects
3. The `this.connected_client` is called, it returns the old client, but this client is no longer connected.

Reply:
Good point. It's obscure, but probably worth shoring up. I'll try to find a way.

Reply:
Updated.